design-editor/src/pane/interaction.js

```diff
@@ -454,19 +454,49 @@ class Interaction {
      * Set resizable
      */
     setResizable() {
-        var options = ElementDetector.getInstance().detect(this._$contentElement).package.options;
-
-        if (options.resizable === true) {
-            if (this._onEditMode) {
-                this._$selectedLayerElement.resizable({
-                    handles: 'e, s, se',
-                    start: this._onResizeStart.bind(this),
-                    resize: this._onResize.bind(this),
-                    stop: this._onResizeStop.bind(this)
+		const resizedElement = this._$contentElement,
+			options = ElementDetector.getInstance().detect(resizedElement).package.options,
+			resizeOptions = options.resizable;
+		let resizingDirectives = '';
+		if (resizeOptions) {
+			if (this._onEditMode) {
+				if (resizeOptions === true) {
+					// set resizing to all directives when resizeOptions is true
+					resizingDirectives = 'e, s, se';
+				} else {
+					// situation when resizeOptions is an object
+					const hasResizeOptionsFalseValues = Object.keys(resizeOptions).some(
```

Comment:
In case if resizable will be defined like this: {"horizontal": true} this will work incorrectly.

I think we can use something like: 

if (resizeOptions['horizontal']) {
  resizingDirectives += 's,'
}
if (resizeOptions['vertical']) {
  resizingDirectives += 's,'
}
if (resizeOptions['horizontal'] && resizeOptions['vertical']) {
  resizingDirectives += 'se,'
}

I have created an issue to change it later. For now it works fine.